project_national/fact_sheets_category_4.yml

```diff
@@ -0,0 +1,318 @@
+      max_crawling_time: 1200
+
+workflow_generator:
+  type: residential_hpxml
+  args:
+    build_existing_model:
+      simulation_control_timestep: 60
+      simulation_control_run_period_begin_month: 1
+      simulation_control_run_period_begin_day_of_month: 1
+      simulation_control_run_period_end_month: 12
+      simulation_control_run_period_end_day_of_month: 31
+      simulation_control_run_period_calendar_year: 2007
+
+    emissions:
+      - scenario_name: LRMER_HighRECost_2030boxavg
+        type: CO2e
+        elec_folder: data/cambium/2022/LRMER_HighRECost_2030boxavg
+        gas_value: 147.3
+        propane_value: 177.8
+        oil_value: 195.9
+
+      - scenario_name: LRMER_LowRECost_2030boxavg
+        type: CO2e
+        elec_folder: data/cambium/2022/LRMER_LowRECost_2030boxavg
+        gas_value: 147.3
+        propane_value: 177.8
+        oil_value: 195.9
+
+      - scenario_name: LRMER_MidCase_2030boxavg
+        type: CO2e
+        elec_folder: data/cambium/2022/LRMER_MidCase_2030boxavg
+        gas_value: 147.3
+        propane_value: 177.8
+        oil_value: 195.9
+
+    utility_bills:
+      - scenario_name: Utility Rates - Fixed + Variable
+        simple_filepath: data/simple_rates/State.tsv
+
+    simulation_output_report:
+      timeseries_frequency: timestep #monthly
+      include_timeseries_total_consumptions: true
+      include_timeseries_fuel_consumptions: true
+      include_timeseries_end_use_consumptions: true
+      include_timeseries_emissions: true
+      include_timeseries_total_loads: false
+
+    reporting_measures:
+      - measure_dir_name: QOIReport
+
+    server_directory_cleanup:
+      retain_in_idf: false
+      retain_schedules_csv: false
+
+### -- Upgrades ---
+upgrades:
+  - upgrade_name: Universal Cooling
+    options:
+      - &add_room_ac
+        option: HVAC Cooling Type|Room AC
+        apply_logic:
+          - and:
+            - HVAC Has Ducts|No
+            - HVAC Cooling Type|None
+      - &add_room_ac_add_27_partial_space_conditioning
+        option: HVAC Cooling Partial Space Conditioning|27% Conditioned
+        apply_logic:
+          - and:
+            - HVAC Has Ducts|No
+            - HVAC Cooling Type|None
+            - Geometry Floor Area Bin|4000+
+      - &add_room_ac_add_36_partial_space_conditioning
+        option: HVAC Cooling Partial Space Conditioning|36% Conditioned
+        apply_logic:
+          - and:
+            - HVAC Has Ducts|No
+            - HVAC Cooling Type|None
+            - Geometry Floor Area Bin|2500-3999
+      - &add_room_ac_add_37_partial_space_conditioning
+        option: HVAC Cooling Partial Space Conditioning|37% Conditioned
+        apply_logic:
+          - and:
+            - HVAC Has Ducts|No
+            - HVAC Cooling Type|None
+            - Geometry Floor Area Bin|1500-2499
+      - &add_room_ac_add_52_partial_space_conditioning
+        option: HVAC Cooling Partial Space Conditioning|37% Conditioned
```

Comment:
Intent document shows that this should be 52% conditioned (floor are less than 1500 sqft)